api-spec/protobuf/operator.proto

```diff
@@ -755,8 +762,19 @@ enum PredefinedPeriod {
   NULL = 0;
   LAST_HOUR = 1;
   LAST_DAY = 2;
-  LAST_MONTH = 3;
-  LAST_3_MONTHS = 4;
-  YEAR_TO_DATE = 5;
-  ALL = 6;
+  LAST_WEEK = 3;
+  LAST_MONTH = 4;
+  LAST_3_MONTHS = 5;
+  YEAR_TO_DATE = 6;
+  ALL = 7;
+}
+
+enum Granularity {
+  HOUR = 0;
+  FOUR_HOURS = 1;
+  DAY = 2;
+  WEEK = 3;
+  MONTH = 4;
+  THREE_MONTHS = 5;
+  YEAR = 6;
```

Comment:
Does a timeframe of three months or entire here makes sense?

Reply:
No it doesn't make sense, I will remove.